### PR TITLE
Add new content installer metadata to RA2.

### PR DESCRIPTION
**Pull request by pchote**
For use with https://github.com/OpenRA/OpenRA/pull/11375.

Closes https://github.com/OpenRA/ra2/issues/124.

### CODE REVIEW

**Comment by Mailaender** [comment]
Works. :white_check_mark:

File: mod.yaml [hunk 1]
```diff
@@ -268,3 +261,36 @@ GameSpeeds:
 		Name: Fastest
 		Timestep: 20
 		OrderLatency: 6
+
+ModContent:
+	InstallPromptMessage: Red Alert 2 requires artwork and audio from the original game.
+	HeaderMessage: The original game content must be copied from an original game disc.
+	Packages:
+		base: Base Game Files
+			TestFiles: ^Content/ra2/ra2.mix, ^Content/ra2/language.mix, ^Content/ra2/multi.mix
```
Comment (abcdefg30) [inline mod.yaml]: We don't need (and probably don't want) `mutli.mix`. It just contains `.map` files.

Comment (pchote) [inline mod.yaml]: @Mailaender: did you have a specific reason for including this in the metadata block you posted on https://github.com/OpenRA/OpenRA/pull/11375?

Comment (Mailaender) [inline mod.yaml]: I use `OpenRA.Utility.exe ra2 --extract` etc. regularly which requires these to be mounted.

Comment (abcdefg30) [inline mod.yaml]: > which requires these to be mounted.

Oo why that.

Comment (Mailaender) [inline mod.yaml]: The command uses the virtual file system. That is independent from this though. You can safely remove it for all the other users.

**Comment by Mailaender** [comment]
Merging this now to fix the build.

**Comment by abcdefg30** [comment]
> Merging this now to fix the build.

👎 If you really desperately need to fix the build you could have just fixed @pchote's commit locally and directly push it (closing this pr), IMHO.

Anyway, filed #236.

**Pull request merged**